app/src/app/app-routing.module.ts

```diff
@@ -1,17 +1,19 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { ProfileComponent } from './modules/main-page/components/profile/profile.component';
 
 const routes: Routes = [
   {
     path: 'start',
     loadChildren: () => import('./modules/landing-page/landing-page.module').then((m) => m.LandingPageModule)
   },
   { path: 'home', loadChildren: () => import('./modules/main-page/main-page.module').then((m) => m.MainPageModule) },
+  { path: 'profile', component: ProfileComponent },
```

Comment:
To trzeba by było przenieść do pliku `main-page-routing.module.ts` (Bo w dokładnie tym module go wywołujemy)

Reply:
`
const routes: Routes = [
  { 
    path: '', 
    component: MainPageComponent, 
    children: [
      { path: 'profile', component: ProfileComponent }
    ] }
];
`